### DISCUSSION TITLE
Spring-data-qldb

### DISCUSSION THREAD

**Discussion by RaposoJoshlan** [awspring/spring-cloud-aws#475] [Q&A]
I want to know whether there is any equivalent of AWS qldb within spring data. eg spring-data-qldb to use for crud operations

**Comment by maciejwalkowiak** [comment]
Not in Spring Cloud AWS, and we currently do not have plans to implement one.

**Reply by RaposoJoshlan** [reply]
Ok. Thanks. Can you suggest any other way of performing crud operations in such case? Or do I have to manually do it.
Thanks in advance

**Reply by maciejwalkowiak** [reply]
As far as I can tell, you need to use the SDK directly.
